### DISCUSSION TITLE
How do you pass in a custom list to fzf?

### DISCUSSION THREAD

**Discussion by kdog3682** [ibhagwan/fzf-lua#1119] [General]
I would like to fuzzy find the list {"apples", "bananas", "grapes"}

I perform the call like this: require('fzf-lua').args(my_list).

However, I get the error "attempt to index local 'k' ( a number value).

Appreciate any help with this!

**Comment by ibhagwan** [comment]
Have you looked at the `fzf_exec` API?

The first argument can take a simple list:
https://github.com/ibhagwan/fzf-lua/wiki/Advanced
